src/LiveApi.js

```diff
@@ -212,6 +212,9 @@ export default class LiveApi {
     }
 
     send(json) {
+        if (Object.keys(json).length === 0 && json.constructor === Object) {
```

Comment:
Did you test with undefined?

Object.keys(undefined) will throw error
I think 

`if (json && Object.keys(json).length === 0) ` would suffice